canonical-web-and-design/jaas.ai
# Add docs module

## Done

Add first draft of docs

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/docs
- make sure docs work
- Open http://0.0.0.0:8029/docs
- Open http://0.0.0.0:8029/docs/commands
- those are custom pages
